src/main/java/com/welovecoding/nbeditorconfig/config/Settings.java

```diff
@@ -5,6 +5,7 @@
 public final class Settings {
 
   public static final String MIME_TYPE = "text/x-editorconfig"; // NOI18N
+  public static final String EXTENSION = "editorconfig"; // NOI18N
```

Comment:
> Lines marked with #NOI18N or other applicable comments on line before a given message/label or group of them denotes what items not to translate for cases where it is not completely obvious that something should not be translated.`

I doubt that we need the `NOI18N` declaration in a settings file because there is no translation for a MIME type, Extension and / or resource path.